centaur/src/main/resources/standardTestCases/localization_sanity_papi_v2/localization_sanity_papi_v2.inputs

```diff
@@ -0,0 +1,11 @@
+{
+  "localization_sanity.files": [
+    "gs://genomics-public-data/platinum-genomes/vcf/NA12877_S1.genome.vcf",
+    "gs://genomics-public-data/platinum-genomes/vcf/NA12892_S1.genome.vcf",
+    "gs://genomics-public-data/platinum-genomes/vcf/NA12889_S1.genome.vcf",
+    "gs://genomics-public-data/platinum-genomes/vcf_deepvariant/0.7.0/NA12878_S1.genome.vcf",
+    "gs://genomics-public-data/platinum-genomes/vcf_deepvariant/0.7.0/NA12892_S1.genome.vcf",
+    "gs://genomics-public-data/platinum-genomes/reports/Variant-Level-QC-DeepVariant-Platinum-Genomes.html",
+    "gs://genomics-public-data/platinum-genomes/reports/Sample-Level-QC-DeepVariant-Platinum-Genomes.html"
```

Comment:
Does it have to be this externally managed bucket, or can we use something cromwell related?

Reply:
I struggled to find something that was sufficiently large and in different subdirectories for this test. How about copying these to a Cromwell bucket?

Reply:
Nah, makes sense. We can simulate large files if/when they're deleted.